posthog/api/plugin.py

```diff
@@ -607,12 +610,15 @@ def job(self, request: request.Request, **kwargs):
         log_activity(
             organization_id=self.team.organization.id,
             # Users in an org but not yet in a team can technically manage plugins via the API
-            team_id=self.team.id,
+            team_id=self.team.pk,
```

Comment:
why this change?

Reply:
pk == id for this model, however we prefer `pk` throughout the codebase.